Cutelyst/response.cpp

```diff
@@ -252,17 +252,15 @@ void Response::redirect(const QUrl &url, quint16 status)
         d->headers.setContentType(QStringLiteral("text/html; charset=utf-8"));
 
         const QString buf = QStringLiteral(
-                    "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0"
-                    "Strict//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\">\n"
+                    "<!DOCTYPE html>\n"
```

Comment:
I've become a performance seeker and I do believe this change make this code a little slower, can you prove me wrong? otherwise I really don't care about readability as this code shouldn't change often nor is client code..

Reply:
you're right. I was not aware of the large differences. I did some small tests with the Google benchmark library. My .arg() is slower by 15% to standard QString concatenation. But you are defining QT_USE_QSTRINGBUILDER (new for me) and that boosts your code to be four times faster. Without the define one could use the %-operator, that would be equally fast.
I will rework my patch.